src/assets/js/modules/sliding-text-animation.js

```diff
@@ -0,0 +1,22 @@
+import { $document, $window } from "../utils/globals";
+
+// Sliding animation for kpi-options page
+$document.ready(() => {
+  function slide() {
+    var curr = 0;
+
+    setInterval(function () {
+      $(`#kpi-cta-container .kpi-hero-cta:nth-of-type(${curr})`).slideUp(
+        "slow"
```

Comment:
was looking at the preview, and the animation does feel a bit slow. might want feedback from @Tommy1231232 and jesper